plugins/XmlMaterialProfile/product_to_id.json

```diff
@@ -12,5 +12,9 @@
     "Ultimaker Original": "ultimaker_original",
     "Ultimaker Original+": "ultimaker_original_plus",
     "Ultimaker Original Dual Extrusion": "ultimaker_original_dual",
-    "IMADE3D JellyBOX": "imade3d_jellybox"
+    "IMADE3D JellyBOX": "imade3d_jellybox",
+    "Strateo3D": "strateo3d",
```

Comment:
This isn't needed. THe id is already strateo3d

Reply:
Ok, see below